### DISCUSSION TITLE
Are the SDK Changelogs broken (or have they moved)?

### DISCUSSION THREAD

**Discussion by johnfedak** [aws/aws-sdk-net#3072] [General]
They do not appear to have been updated since build 3.7.660 on 6-Oct

https://github.com/aws/aws-sdk-net/blob/master/changelogs/SDK.CHANGELOG.2023.md

**Comment by boblodgett** [comment]
They are updated here: https://github.com/aws/aws-sdk-net/blob/main/changelogs/SDK.CHANGELOG.2023.md

The problem is the link you are using is using the `master` branch when we moved to `main` as the default branch. Here is the announcement: https://github.com/aws/aws-sdk-net/issues/3066

If you got the link from a specific page or reference please let us know so we can correct the link.

**Reply by drewburlingame** [reply]
The link to the changelon in the main readme points to master.

**Reply by normj** [reply]
Thanks for letting us know. I created a PR to update the README to point to `main`. https://github.com/aws/aws-sdk-net/pull/3396

**Comment by johnfedak** [comment]
Nope, had it bookmarked and missed the announcement.  Thanks.
